ert3/config/_parameters_config.py

```diff
@@ -76,6 +76,27 @@ def _ensure_lower_upper(cls, values):  # type: ignore
         )
 
 
+class _LogUniformInput(_ParametersConfig):
+    lower_bound: float
+    upper_bound: float
+
+    @root_validator
+    def _ensure_lower_upper(cls, values):  # type: ignore
+        low = values.get("lower_bound")
+        up = values.get("upper_bound")
+
+        if low is None or up is None:
+            return values
```

Comment:
I did not notice that since I basically the code, but I think this because a pydantic root validator is used. If you do that you have to be prepared that some values are None, due to the occurrence of an validation error before this validator is called. For instance, if you provide a value that cannot be parsed to a float, that causes a validation error before this root validator is called. The root validator will still be called, with a None for the offending value.